### PR TITLE
create an imagestream import secret for importing samples

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: roles/openshift_examples/tasks/main.yml [hunk 1]
```diff
@@ -49,6 +49,18 @@
 ######################################################################
 # Begin image streams
 
+- name: Create imagestream import secret
+  command: >
+    {{ openshift_client_binary }} create secret docker-registry imagestreamsecret --docker-server={{ registry_host }} --docker-username={{ oreg_auth_user }} --docker-email=openshift@openshift.com --docker-password={{ oreg_auth_password }} --config={{ openshift.common.config_base }}/master/admin.kubeconfig -n openshift
+  when:
```
Comment (vrutkovs) [inline roles/openshift_examples/tasks/main.yml]:
```
roles/openshift_examples/tasks/main.yml:
  55:8      error    trailing spaces  (trailing-spaces)
```

Comment (bparees) [reply roles/openshift_examples/tasks/main.yml]: thanks, fixed.

**Comment by bparees** [comment]
[(5) Tolerate auth-enabled redhat registry [builds]](https://trello.com/c/Howwzs2c/1492-5-tolerate-auth-enabled-redhat-registry-builds)

**Pull request merged**